l5kit/l5kit/environment/reward.py

```diff
@@ -0,0 +1,91 @@
+from typing import DefaultDict, Dict, List, Optional
+
+import numpy as np
+import torch
+
+from l5kit.simulation.dataset import SimulationDataset
+from l5kit.simulation.unroll import UnrollInputOutput
+from l5kit.environment.cle_metricset import L5BaseMetricSet, SimulationOutputGym
+
+
+class Reward(object):
```

Comment:
Please create an abstraction here: you will have `Reward` (that can be an ABC) and then you will have something like a `L2ImitationReward` (this concrete implementation of the `Reward`). Imagine that can exist multiple rewards later.